e2e/typescript-coverage/typescriptPreprocessor.js

```diff
@@ -8,18 +8,19 @@
 const tsc = require('typescript');
 
 module.exports = {
-  process(src, path) {
-    if (path.endsWith('.ts') || path.endsWith('.tsx')) {
-      return tsc.transpile(
-        src,
-        {
+  process(sourceText, fileName) {
+    if (fileName.endsWith('.ts') || fileName.endsWith('.tsx')) {
+      const {outputText, sourceMapText} = tsc.transpileModule(sourceText, {
```

Comment:
why the change from `transpile` to `transpileModule`?

Reply:
`transpile` returns only a string. No source map. In a way this test was broken. It was supposed to test coverage, but that can hardly work correctly without a source map. The test was passing, because coverage was 100%.

Reply:
aha, fair enough!